packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/taskService.ts

```diff
@@ -126,7 +126,22 @@ class TaskService {
       sfdxTaskId: taskId
     };
     const taskSource = 'SFDX';
-    const taskShellExecution = new vscode.ShellExecution(cmd, args);
+    // https://github.com/forcedotcom/salesforcedx-vscode/issues/2097
```

Comment:
I think this should be ok, might want to add posting a message to the output panel saying we're defaulting to cmd.exe so the user that's on git bash or other client is aware of it. 

@mysticflute, @smaddox-sf what do you think ?

Reply:
If I understand the issue, this does not occur by default on windows but only if the user has specifically set the default shell to git bash? If that's correct then it sounds reasonable to output a message

Reply:
@mysticflute That's correct. But we don't know if the user has set the default shell to git bash or cmd.exe. If it's fine outputting a message every time in Windows, I'll add it. Frankly I think this won't actually affect the experience, since after running a test the terminal would not be reusable for the user. (The terminal would say: "Terminal will be reused by tasks, press any key to close it." )

Reply:
I see. Is it possible to print to the output panel without forcing it to display/pop up? If the message can be added in the background without disrupting the user each time that sounds better to me.